### PR TITLE
Add pytest-cpp

### PR DESCRIPTION
**Pull request by gqmelo**

### CODE REVIEW

**Comment by gqmelo** [comment]
cc: @nicoddemus

File: recipes/pytest-cpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pytest
```
Comment (jakirkham) [inline recipes/pytest-cpp/meta.yaml]: Is this needed during build time or just run time?

File: recipes/pytest-cpp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+about:
+  home: http://github.com/pytest-dev/pytest-cpp
+  license: MIT
+  license_file: LICENSE
+  summary: "Use pytest's runner to discover and execute C++ tests"
+
+extra:
+  recipe-maintainers:
+    - gqmelo
+    - nicoddemus
```
Comment (jakirkham) [inline recipes/pytest-cpp/meta.yaml]: Is it alright for you to be a maintainer here, @nicoddemus ?

Comment (nicoddemus) [inline recipes/pytest-cpp/meta.yaml]: Yes, thanks for asking! :grin:

File: recipes/pytest-cpp/meta.yaml (hunk 1 above)
Comment (nicoddemus) [inline recipes/pytest-cpp/meta.yaml]: FWIW other pytest-plugins have the same dependency during build time:

- [pytest-faulthandler](https://github.com/conda-forge/pytest-faulthandler-feedstock/blob/master/recipe/meta.yaml#L22)
- [pytest-mock](https://github.com/conda-forge/pytest-mock-feedstock/blob/master/recipe/meta.yaml#L22)
- [pytest-qt](https://github.com/conda-forge/pytest-qt-feedstock/blob/master/recipe/meta.yaml#L22)
- [pytest-xdist](https://github.com/conda-forge/pytest-xdist-feedstock/blob/master/recipe/meta.yaml#L23)
- [pytest-benchmark](https://github.com/conda-forge/pytest-benchmark-feedstock/blob/master/recipe/meta.yaml#L20)

But on other hand it doesn't really seem necessary:

- [pytest-cov](https://github.com/conda-forge/pytest-cov-feedstock/blob/master/recipe/meta.yaml#L17)

Comment (gqmelo) [inline recipes/pytest-cpp/meta.yaml]: Indeed it is not necessary during build time. I removed it

**Comment by gqmelo** [comment]
@jakirkham, could this be merged?

**Comment by jakirkham** [comment]
Yep, sorry, thanks for reminding me. 😄

**Pull request merged**